sdks/typescript/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "@ethereal-pulse/typescript-sdk",
   "version": "0.0.1",
-  "type": "module",
```

Comment:
So i didn't remove it, because I'm not 100% sure that i can do it, because form what i understood commonjs still is the "format" for the old browsers. For now, i decided to let it as is,  decide that on next steps.

Reply:
@danielfernandes2022 I just noticed that since we are exporting commonjs we can keep `type: module` to be used inside our packages